src/pages/infrastructure.js

```diff
@@ -94,6 +94,17 @@ const emergingProjects = [
       {label: 'Slack channel', url: 'https://cilium.slack.com/archives/C0210QTK2MV'},    
     ],
   },
+  {
+    name: 'uBPF',
+    title: 'uBPF - Userspace eBPF Runtime',
+    description: `An extended Berkley Packet Filter runtime that permits execution of
```

Comment:
```suggestion
    description: `An eBPF runtime that permits execution of
```
eBPF no longer stands for anything